fix(language_server): don't apply "ignore this rule" fixes for fixAll code action + command

Sysix opened the pull request:
This PR fixes a bug where on `source.fixAll.oxc` code action or `oxc.fixAll` command will append `// oxlint-disable ...` comments to the file.
This bug was introduced in https://github.com/oxc-project/oxc/pull/14183

Sysix commented on crates/oxc_language_server/src/code_actions.rs:
```diff
@@ -87,7 +88,29 @@ pub fn apply_all_fix_code_action<'a>(
             // Applying all possible fixes at once is not possible in this context.
-            PossibleFixContent::Multiple(multi) => multi.first(),
+            PossibleFixContent::Multiple(multi) => {
+                if multi.len() > 2 {
+                    multi.first()
```
no, there should be "fix rule" "disable for this line" and "disable for this file"

camc314 commented:
### Merge activity

* **Sep 29, 8:37 PM UTC**: The merge label '0-merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=oxc-project&repo=oxc) once it meets the requirements.

Sysix committed:
fix(language_server): don't apply "ignore this rule" fixes for fixAll code action + command (#14243)

This PR fixes a bug where on `source.fixAll.oxc` code action or `oxc.fixAll` command will append `// oxlint-disable ...` comments to the file.
This bug was introduced in https://github.com/oxc-project/oxc/pull/14183
